source/swap/package.json

```diff
@@ -27,16 +27,15 @@
     "verify": "truffle run verify"
   },
   "devDependencies": {
-    "@airswap/tokens": "0.1.3",
     "@airswap/order-utils": "0.3.16",
     "@airswap/test-utils": "0.1.3",
-    "truffle": "^5.0.32",
-    "@gnosis.pm/mock-contract": "^3.0.7",
+    "@airswap/tokens": "0.1.3",
+    "openzeppelin-solidity": "2.4",
```

Comment:
You'd added `openzeppelin-solidity` here but it's already in the `dependencies` section. I think it can be removed from there now as it's now just a `devDependencies`